### PR TITLE
[ADDED] Section, SectionItems and SectionSectionItemsThrough models

### PR DESCRIPTION
**Pull request by rinkishivani**

### CODE REVIEW

File: products/admin.py [hunk 1]
```diff
@@ -23,3 +22,23 @@ class Brand(admin.ModelAdmin):
 @admin.register(BrandProduct)
 class BrandProduct(admin.ModelAdmin):
     list_display = ('brand', 'product')
+
+
+@admin.register(Section)
+class Section(admin.ModelAdmin):
+    list_display = ('name', 'order', 'is_active', 'section_file')
+
```
Comment (snehals21) [inline products/admin.py]: @rinkishivani give proper name to admin classes. e.g SectionAdmin, SectionItemsAdmin, etc. and add inline admin class for through model.
reference: https://medium.com/@aswinshenoy/django-admin-setting-up-many-to-many-relationships-with-intermediate-models-782895cb48b2c4c60c6d57a7
cc: @lalitvasoya

Comment (rinkishivani) [reply products/admin.py]: Done

File: products/admin.py [hunk 2]
```diff
@@ -1,25 +1,50 @@
...
+    list_display = ('section_items', 'section')
+
+
+@admin.register(Banner)
+class BannerAdmin(admin.ModelAdmin):
+    list_display = ('banner_name', 'banner_image', 'is_active', 'created_date', 'updated_date')
+
+
+class SectionItemsInline(admin.TabularInline):
+    model = Section.section_items.through
```
Comment (snehals21) [inline products/admin.py]: @rinkishivani here use through model instead of taking manytomany field.

cc: @lalitvasoya

Comment (rinkishivani) [reply products/admin.py]: Done

**Review by snehals21 — APPROVED** [review]

**Pull request merged**